include/maliput/routing/find_lane_sequences.h

```diff
@@ -56,5 +56,11 @@ std::vector<std::vector<const maliput::api::Lane*>> FindLaneSequences(const mali
                                                                       const maliput::api::Lane* end,
                                                                       double max_length_m);
 
+/// Overload to @ref FindLaneSequences() that removes from the result those
+/// sequences presenting U-turns when @p remove_u_turns is true. Otherwise,
+/// the function behaves exactly the same.
+std::vector<std::vector<const maliput::api::Lane*>> FindLaneSequences(const maliput::api::Lane* start,
+                                                                      const maliput::api::Lane* end,
+                                                                      double max_length_m, bool remove_u_turns);
```

Comment:
I wonder whether "remove_u_turns" should instead be "no_u_turns." The "remove" reveals implementation detail that the user probably doesn't need to know. Specifically, the user need not know that the implementation first obtains all routes including those with u-turns, and then subsequently filters out the ones with u-turns. The user only needs to know that the result does not have u-turns, hence the recommendation to use parameter name "no_u_turns."

Reply:
Done in c908cba.